server/app/assets/javascripts/preview.ts

```diff
@@ -77,6 +78,7 @@ class PreviewController {
   private static accordionContent = '>'
   private static accordionHeader = '### '
   private static bulletedItem = '* '
+  private static headerIndicator = '#'
```

Comment:
What's this do?

Reply:
`renderInline` and the corresponding custom Java method I wrote `renderWithoutWrappingPTags` trim the wrapping tags off the block, regardless of what they are. This meant that header tags were getting trimmed and headers weren't showing up as expected. So I added this var + the `if` check in `parseMarkdown` to get around that. If it detects a header, it calls the regular `render` method that doesn't trim the wrapping tags. It's kinda funky and I'm open to other ideas of how to handle this if you have them.

Reply:
Makes sense. It did just occur to me that we probably should not allow for an H1 to be added here since there should only ever be a single H1 on a page. 

We even log it if there are [more than one found](https://github.com/civiform/civiform/blob/6339bd864ed472cbabfc86b7b0563e4a4075c754/server/app/views/applicant/ApplicantLayout.java#L113), which I haven't been good at reporting.

Reply:
Oh good call! I could add some logic that automatically turns it into `##` or just trims off the H1. I need to check how big/small that allows us to make the text in that case, since that's a requirement. My other idea was to write some custom logic to apply the tailwind text classes but would be great to not have to write more custom logic. What do you think?